### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by Arizar98**

### CODE REVIEW

**Comment by Arizar98** [comment]
Ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
+
```
Comment (saobv-2223) [inline Gemfile]: use " please

File: app/mailers/application_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
```
Comment (saobv-2223) [inline app/mailers/application_mailer.rb]: using "

File: config/credentials.yml.enc [hunk 3]
```diff
@@ -0,0 +1 @@
+rZnsNJXszaYhS95au6gMs9jLs6sMReyzm9u93Cfbcd2Os9TekUfdbgfe+uFMVWFdUmpJq84TdTXCNHpUY0KhK1E+vMAtIX5KL0MykVuqMy28LeC3B7KjP2n6JF7O/z4NUBW25MfEx/UTEQWR7oipRa/aIj34NmoD5CJ7IYYvxdoNl877VSqefhlIOofpR/XEu+RLoZT9YT/GUWjYYBLkEKk5GakplKqzTjCOTqkPJchI5K92Tluukb+HFjNLER05JiUiBpopZtVyZDpO/IYG6lOk6wFbY4yskg8zawd9xMnN0m00vd9SForaOiPwznYDTTZ6gu/yiLJzJOvM2UIohCHXlGvrlRzwClyJrJSvwAbJNSHw0A9pNTaTdi+2Rave+yERzaNev4+uHTgN4UYIIf1HqGtcCG0kbR+5--K6DoXjWzJUuAW7hM--GAco5RzmMZNdmxNh4unJ9w==
```
Comment (saobv-2223) [inline config/credentials.yml.enc]: create `credentials.yml.enc.example` and push to git, this file you should add to `.gitignore`

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (saobv-2223) [inline config/database.yml]: do same as credentials.yml.enc

File: test/application_system_test_case.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
```
Comment (saobv-2223) [inline test/application_system_test_case.rb]: add test to gitignore, we dont use it in project

**Comment by Arizar98** [comment]
Ready

**Review by saobv-2223 — COMMENTED** [review]
Your pull is having 4 commits, merge to only 1 commit.
Follow some common mistakes and fix them.
Then ready again
https://docs.google.com/document/d/17CBB-BCQm6wu60cCiAmYf7tibU-VaKLe1TytCU9wSVc/edit?usp=sharing

**Comment by Arizar98** [comment]
Ready

**Review by saobv-2223 — COMMENTED** [review]
Hi, why you pull request only have 10 files changed ?

**Comment by Arizar98** [comment]
> Hi, why you pull request only have 10 files changed ?

Because I was fixing the previous comments, and I realised that I did not do the "mysql2" properly during the first push.

**Comment by Arizar98** [comment]
Ready

File: app/mailers/application_mailer.rb [hunk 6]
```diff
@@ -1,4 +1,6 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
-  layout 'mailer'
+
```
Comment (saobv-2223) [inline app/mailers/application_mailer.rb]: remove empty line

File: config/boot.rb [hunk 7]
```diff
@@ -1,4 +1,6 @@
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
-require 'bundler/setup' # Set up gems listed in the Gemfile.
-require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
+ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)
+
+require "bundler/setup" # Set up gems listed in the Gemfile.
+require "bootsnap/setup" # Speed up boot time by caching expensive operations.
+
```
Comment (saobv-2223) [inline config/boot.rb]: image

**Comment by Arizar98** [comment]
Ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**